st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -376,7 +376,7 @@ def get_all(self, exclude_attributes=None, **kw):
         if 'timestamp_lt' in kw:
```

Comment:
For explicitness sake, I would also add `or 'sort_desc' in kw:` here so it's consistent with the elif clause below.